django_north/management/commands/migrate.py

```diff
@@ -42,10 +42,7 @@ def handle(self, *args, **options):
 
```

Comment:
> Remove callbakc usage

I think there's a typo in the commit title. It's not important at all, but if you rebase you may as well fix it.